### ISSUE TITLE
BitmapButton bool not firing propertyManagerPage DataChanged Event

### ISSUE THREAD

**Issue by emersonbottero** [xarial/xcad#20]
``` C#
        [Description("Barra Quadrada")]
        [BitmapButton(typeof(Resources), nameof(Resources.Barra_Quadrada), 48, 48)]
        [ControlOptions(left: 37, top: 0)]
        public bool SquareBar { get; set; } = false; 
```
``` C#
        private void M_Page_DataChanged()
        {
            var test = m_Data;
            
                //TODO: handle the data changing, e.g. update preview
        }
````

if I set a breakpoint on var test = ... and click on the button the breakpoint is never reached
On combobox change the breakpoint is reached

**Comment by emersonbottero** [comment]
is it available on the last dev Nugget Package?

**Comment by artem1t** [comment]
That is already in master and published to the public 0.6.1 nuget package
